### PR TITLE
build: golangci-lint v1.53.2

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -10,25 +11,21 @@ jobs:
       matrix:
         go: ['1.19', '1.20']
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab #v3.5.2
```
Comment (davecgh) [inline .github/workflows/build.yml:14]: Confirmed v3.5.2 is 8e5e7e5ab8b370d6c329ec480221332ada57f0ab.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -10,25 +11,21 @@ jobs:
...
 
     - name: Set up Go
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 #v3.5.0
```
Comment (davecgh) [inline .github/workflows/build.yml:17]: Confirmed v3.5.0 is 6edd4406fa81c3da01a34fa6f6343087c207a568.

However, v4.0.1 it out.  Any reason not to just use the latest?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -39,10 +36,10 @@ jobs:
       matrix:
         node-version: [16.x, 18.x]
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab #v3.5.2
 
       - name: Use nodejs ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c #v3.6.0
```
Comment (davecgh) [inline .github/workflows/build.yml:42]: Confirmed v3.6.0 is 64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c.

**Comment by davecgh** [comment]
Commit and PR descriptions no longer match the code since it was updated to v1.53.2.

**Review by chappjc — APPROVED** [review]

**Pull request merged**